### DISCUSSION TITLE
Ny komponent: Toast

### DISCUSSION THREAD

**Discussion by onordbo** [fremtind/jokul#2816] [Annet]
BM har implementert en toast-funksjon via MessageBox. I forum ble vi enige om at dette ikke nødvendigvis er et snøfnugg, og ønsker at teamene samler og legger ut eksempler i sine tjenester og løsninger hvor de kan ha behov for dette. Legg det gjerne ut her, og diskuter videre 👋 Vi tar en kikk på alt som har blitt samlet og diskutert ved neste designsystem-forum (05/04).

Vi burde kanskje også finne ut av, og bli enige om i hvilke situasjoner vi ønsker å bruke toast (Ex: hendelser vs handlinger) osv?

**Comment by onordbo** [comment]
@Murstam: Del gjerne litt av det dere presenterte i forum her 😄

**Comment by Murstam** [comment]
Her er eksempelet fra kundeløsning BM som jeg viste på DS-forum.

<img width="2140" alt="Skjermbilde 2022-03-30 kl  09 40 26" src="https://user-images.githubusercontent.com/47025023/160777971-36b90bb4-aaa7-41d6-8cd1-5664e9e9a0c1.png">

**Comment by ghost** [comment]
Her er eksempelet eg nevnte fra SeOpp. I forum vart bruksmønsteret "gi informasjon som gjelder noko som ikkje fins" nevnt. Sånn sett hadde det passa å bruke toast her.

Sånn det er i dag har me hverken bekreftelse av sletting, eller indikasjon på at slettingen har skjedd.

![image](https://user-images.githubusercontent.com/55135175/162174999-f3537f89-fb93-4991-a560-6ae2380a1e33.png)

**Comment by ghost** [comment]
Dette er ein personlig meining: Eg likar godt når eg får angre ein slette-operasjon i ein toast. Sjå dette eksempelet frå Google Drive:

https://user-images.githubusercontent.com/89779105/162177368-5030ceb7-b8d2-4ce2-b23c-6045917d6ca3.mov

**Comment by ivnai** [comment]
Så ikke denne før :/ 
Her er 2 eksempler hvor brukere bør se bekreftelse på action. Det er flere så vi er happy at den lages!
Litt usikker på hvor vi trenger/kan angre option nå.
![Screenshot 2023-04-04 at 15 07 11](https://user-images.githubusercontent.com/93194579/229801726-6f5269fd-5592-405c-a1e9-71498109fd5c.png)
![Screenshot 2023-04-04 at 15 07 24](https://user-images.githubusercontent.com/93194579/229801758-337489fa-ab77-4301-85df-ce4d869ea561.png)
